Packs/RecordedFuture/Integrations/RecordedFuture/RecordedFuture.yml

```diff
@@ -63,11 +63,30 @@ configuration:
   defaultvalue: "false"
   type: 8
   required: false
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Rule names to fetch alerts by, separated by semicolon. If empty, all alerts
+    will be fetched
+  name: rule_names
+  required: false
+  type: 12
+- defaultvalue: 24 hours
+  display: First fetch time (<number> <time unit>, e.g., 12 hours, 7 days, 3 months,
+    1 year)
+  name: triggered
```

Comment:
```suggestion
  name: fetch_time
```

Reply:
demisto validator saying to set this to `first_fetch`
and found this doc
https://xsoar.pan.dev/docs/tutorials/tut-design#fetch-incidents-parameters
so set the name to  `first_fetch`